setup/tinyHIPPO_populate_equations.sql

```diff
@@ -0,0 +1,56 @@
+BEGIN TRANSACTION;
```

Comment:
So we are only shipping six equations with tinyHIPPO?

Reply:
It won't be hard at all to add more, but I wanna see memory impact first before then. But that's the majority of the traffic types that the IoT traffic falls into

Reply:
If 6 basic traffic equations/signatures don't eat up too much I can add more, but it'd only get more intensive the more specific we get.

Reply:
Hey more graphs!!!! After this is merged we should make an "intensity-testing" branch and add ~20-40 equations and see the impact. That will inform our testing output and future growth

Reply:
We could do that. After this and my mobile project, I'll work on adding filters to the equations and then I can make specific equations for anything captured in the DB. But for each one of those there will likely be an increased memory cost since the entire object will be loaded into memory constantly. But definitely something we can test

Reply:
I agree that we should see how adding more and more equations will affect performance. We might not see too much impact on the Netgear router since we are pretty close to the memory cutoff, but we should be able to observe a difference (if any) on the Linksys

Reply:
Could the logic in this file just be moved to the db setup file? Since it runs anytime tinyHIPPO starts anyways

Reply:
Functionally that would work, but if we plan to ever allow users to import there own equations, it would make sense to have the static DB creation SQL execution separate from the inserts that would be affected by user input.